File: fixtures/go_output/get_with_all_numerical_data.go
package main

import (
	"fmt"
	"io/ioutil"
	"log"
	"net/http"
)

func main() {
	client := &http.Client{}
	var data = []byte(`{18233982904}`)
	req, err := http.NewRequest("POST", "http://198.30.191.00:8309/CurlToNode", data)
	if err != nil {
		log.Fatal(err)
	}
	req.Header.Set("Content-Type", "application/json")
	req.Header.Set("Accept", "application/json")
	resp, err := client.Do(req)
	if err != nil {
		log.Fatal(err)
	}
	bodyText, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("%s\n", bodyText)
}

File: test/fixtures/javascript-jquery/get_with_all_numerical_data.js
$.ajax({
  url: 'http://localhost:28139/CurlToNode',
  crossDomain: true,
  method: 'post',
  headers: {
    'Accept': 'application/json'
  },
  contentType: 'application/json',
  data: '18233982904'
}).done(function(response) {
  console.log(response);
});
